linkerd/announcer/curatorsd/src/main/scala/com/medallia/l5d/curatorsd/announcer/CuratorSDAnnouncer.scala

```diff
@@ -43,6 +44,9 @@ class CuratorSDAnnouncer(zkConnectStr: String) extends FutureAnnouncer {
 
     val serviceInstance = builder.build
 
+    serviceDiscoveryInfo.curatorClient.create().creatingParentContainersIfNeeded().withMode(CreateMode.PERSISTENT).
```

Comment:
Containers is a feature added in ZK 3.5, which is not stable yet. Does this work with earlier versions?